web/src/components/dateRangePicker/DateRangePickerWrapper.tsx

```diff
@@ -30,14 +31,14 @@ const DateRangePickerWrapper: React.FunctionComponent<DateRangePickerWrapperProp
   const handleKeyPress = (event: any) => event?.key === 'Escape' && handleToggle()
 
   const minDate = initialDateRange.startDate || new Date()
-  const maxDate = DateTime.fromJSDate(minDate).plus({ days: 6 }).toJSDate()
+  const maxDate = DateTime.fromJSDate(minDate).plus({ days: 10 }).toJSDate()
 
   return (
     <div data-testid="date-range-picker-wrapper">
-      {open && <div onKeyPress={handleKeyPress} onClick={handleToggle} />}
+      {open && <div onKeyDown={handleKeyPress} onClick={handleToggle} />}
```

Comment:
I can't seem to hit the `onClick` handler for this div, probably because it has no height. I think that can be removed. Could the `onKeyDown` handler moved to the parent div and have `handleKeyPress` include the check for `open`
`const handleKeyPress = (event: any) => open && event?.key === 'Escape' && handleToggle()`

Reply:
Nice call, fixed in: https://github.com/bcgov/wps/pull/2709/commits/99a3d48c9a38831fd667f991eb2228145048e384